reactioncommerce/reaction
# Fix shipping records w/o shopId

Add shopId to all shipping records and removes many (but far from all) of the `shipping[0]` references.

This looks like a larger PR than it is because I migrated the `copyCartToOrder` method to its own file.

**Note** It should be super awesome if we could have Simple-Schema enforce that all shipping records need to contain a shopId but I could not get this to work. (I removed the `blackbox` record to make it validate, but it threw all sort of errors). I don't think this should block us now but I would like to revisit this as soon as we make the change to `node-simple-schema`.

### To Test

1. Create multiple shops
1. Add an item from each shop
1. Observe that there is a shipping records created for each shop that contains the shopId
